src/ItemsDialog.test.js

```diff
@@ -33,6 +33,9 @@ jest.mock('./components', () => ({
 }));
 
 describe('ItemsDialog', () => {
+  const testIds = {
+    loading: 'loading',
+  };
```

Comment:
Could you move it above `describe` to be consistent with test writing approaches?

Reply:
Fixed